src/client/src/containers/RightSidebar/styles.module.css

```diff
@@ -12,6 +12,10 @@
   overflow: auto;
 }
 
+.rightViewCroppedSummaryPage {
+  width: 400px !important; /*change this width after top nav bar migration to change the width of the right side bar on summary page*/
+}
+
```

Comment:
Did you mean to include style changes as part of this PR? I thought this PR is just about removing the instance of right sidebar from the home page.
Is this including the change for `greater width` of right side-bar on summary page?